src/components/Community/Post.tsx

```diff
@@ -43,14 +42,24 @@ export interface PostType {
   comments: number;
   stadium: string;
   weather: string;
-  result: string;
+  result: TeamScheme;
   photo: number;
 }
 
 interface Props {
   post: PostType;
+  scheme?: TeamScheme;
 }
 
+const CHEER_TEAM_STYLE =
+  "inline-flex items-center rounded-lg px-1 py-0.3 text-sm font-bold text-white";
+
+const CheerTeamBadge = ({ scheme }: { scheme?: TeamScheme }) => {
+  if (!scheme) return null;
```

Comment:
if 조건식은 한줄이라고 해도 중괄호를 사용해주는 걸 일반적으로 권장하기는 해요~
```suggestion
  if (!scheme) { 
    return null;
  }
```